NEWS.md

```diff
@@ -127,9 +130,10 @@ Breaking changes
     If a reshaped copy is needed, use `copy(reshape(a))` or `copy!` to a new array of
     the desired shape ([#4211]).
 
-  * `mapslices` now re-uses temporary storage. Recipient functions
-    that expect input slices to be persistent should copy data to
-    other storage ([#17266]).
+  * `mapslices` now re-uses temporary storage. Recipient functions that expect
```

Comment:
Still accurate unless there's something newer than #17266. What got changed is that the new one makes it impossible to overwrite the input array.